ddmore.Rmd

```diff
@@ -20,10 +20,23 @@ data_dir <- "./testdata/nmoutputs"
 file_paths <- list.files(data_dir, "\\.lst$", full.names = TRUE)
 bdf <- function(.filename){
   df <- data.frame( stringsAsFactors = FALSE,
-                    filename = as.character(basename(.filename)),  
-                    lines = readr::read_lines(.filename)) 
+                    file_name = as.character(basename(.filename)),  
+                    line_text = readr::read_lines(.filename))  %>%    
+                        mutate (line_number = row_number()) %>% 
+                          select(file_name, line_number, line_text)
```

Comment:
what's the point of this final select? seems these are they only columns anyway? From looking it seems you're actually just trying to reorder the columns so line number is second, not last. If so, one thing you can do so you don't accidentally start dropping columns if you end up adding more later is to do `select(file_name, line_number, everything()`, which will auto add every column after

Reply:
ok